tests/test_utils.py

```diff
@@ -803,3 +803,43 @@ def set_bootstrap_random_seed_variable(
         rule.variables = build_parameter_container_for_variables(
             variables_configs=variables_dict
         )
+
+
+def find_strings_in_nested_obj(obj: Any, target_strings: List[str]) -> bool:
```

Comment:
@cdkini What can (or should?) we do about the following two questions:
1) We have `find_code_in_notebook()` and `replace_code_in_notebook()` in our `CLI` rendering utils.  Should these utilities be combined, or should they remain on their own?
2) When I was developing the above, if `find_strings_in_nested_obj()` existed and I had known about it, I would have attempted to leverage it.  What can we do towards making our team members aware of the useful utilities (for practical purposes as well as for avoiding re-implementing the same logic)?  Happy to brainstorm.  Thanks! /cc @donaldheppner

Reply:
Maybe a README that documents the above?

Reply:
re #2, that is why I don't like utility methods, they are undiscoverable. :P